### DISCUSSION TITLE
UnDiscord BetterDiscord Plugin

### DISCUSSION THREAD

**Discussion by Fvndei** [victornpb/undiscord#382] [3. Ideas] [answered]
How about making the "UnDiscord" extension to a BetterDiscrod plugin?

**Comment by victornpb** [comment, marked as answer]
its been worked on here https://github.com/victornpb/undiscord/pull/357

**Reply by victornpb** [reply]
Work has been paused, I haven't found a way to automatically get the token in the electron app. Might be impossible, so the only resort in that case would be to add the token text field back, and the user have to manually find it and put it there, But it's so much more work to explain that is easier to just tell people to use the web version.

**Reply by dekrypted** [reply]
Getting the token is easy! (I think)

```js

const electron = require("electron")

var window = electron.BrowserWindow.getAllWindows()[0]
var token = window.webContents.executeJavaScript("(webpackChunkdiscord_app.push([[''],{},e=>{m=[];for(let c in e.c)m.push(e.c[c])}]),m).find(m=>m?.exports?.default?.getToken!==void 0).exports.default.getToken()", true)
```

I haven't tested it but hopefully that works?

**Reply by victornpb** [reply]
oh, I will have to test this. But thank you for commenting that.

**Comment by dekrypted** [comment]
How can I use this though?
